### DISCUSSION TITLE
Trying to work with React Native

### DISCUSSION THREAD

**Discussion by tpmai22** [supabase/supabase#6018] [Questions]
Hi I'm trying to jump into using `Supabase` to store some data for my project which have this structure 
![image](https://user-images.githubusercontent.com/43376325/159622594-f59be7cf-a5bd-4e20-bc38-79c71bb4b615.png)
 I got my `supabase_client.js` setting up like this. I use `babelplugin-inline-dotenv` to work with `.env` file

```js
import AsyncStorage from '@react-native-async-storage/async-storage';
import { createClient } from '@supabase/supabase-js';

const ANON_KEY = process.env.ANON_KEY;
const SUPABASE_URL = process.env.SUPABASE_URL;

const supabase = createClient(SUPABASE_URL, ANON_KEY, {
  localStorage: AsyncStorage,
  detectSessionInUrl: false,
});

export default supabase;
```

I try to query for it but I recieve no return after the query

```js
import supabase from '../../api/supabase_client';
...

 const [studentQuote, setStudentQuote] = useState([]);

async function fetchStudentQuotes() {
    const { data } = await supabase.from('quotes').select('*');
    console.log(data);
    setStudentQuote(data);
  }

  useEffect(() => {
    fetchStudentQuotes();
    // setStudentQuote(studentQuote[Math.floor(Math.random() * studentQuote.length)]);
  }, []);
```

Would some `supadev` can direct me on what might went wrong with my work please ?

**Comment by GaryAustin1** [comment]
Do you have RLS on that table, if so is it set to allow anon on select (unless you are signing in somewhere else)?
Is data coming back null or and empty array?  Not sure what "no return" means.
Any errors in the network request?  You could add {data,error} to get a bit more info potentially.

**Comment by tpmai22** [comment]
it did not have RLS on that table and I try also `console.log(error,'error')` but receive none in return. I could not know whether the data coming back `null` or `empty`

**Comment by tpmai22** [comment]
@GaryAustin1 Is it because we using `Docker` to self host it ? 
![image](https://user-images.githubusercontent.com/43376325/159838597-f6a97ea3-a75a-4f24-b5ca-9b3f67d65f34.png)

**Comment by tpmai22** [comment]
I got something back 
![image](https://user-images.githubusercontent.com/43376325/159840558-390d49f7-99fd-4476-9260-e9a2b11a0b9a.png)

**Reply by GaryAustin1** [reply]
Yeah it sounds like you are not connected up correctly so there is no server to take the request.  
I am not familiar enough with docker set ups to make suggestions.
I would start a new question on your docker setup not working.
